components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -189,7 +189,7 @@ const UserMenuRender: React.ForwardRefRenderFunction<unknown, UserMenuProps> = (
                             (item: UserMenuItem, itemIndex: number): NavItem =>
                                 Object.assign({ itemID: itemIndex.toString() }, item, {
                                     InfoListItemProps: Object.assign(
-                                        { iconColor: theme.palette.text.secondary },
+                                        { iconColor: group.iconColor || theme.palette.text.secondary },
```

Comment:
As it turns out, the individual menu items also take a color prop for the icon that they inherit from the DrawerNavItem component: `itemIconColor`. So we will actually want to account for that as well:
```suggestion
                                        { iconColor: UserMenuItem.itemIconColor || group.iconColor || theme.palette.text.secondary },
```